partcad/tests/unit/test_git_clone_retries.py

```
@@ -104,7 +115,7 @@ def side_effect(*args, **kwargs):
     assert mock_clone_from.call_count == test_git_retry_config["max"] + 1
 
 
-def test_project_import_git_clone_retry_then_success():
+def test_project_import_git_clone_retry_then_success(git_config):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_project_import_git_clone_retry_then_success`

Add return type annotation: `None`

(ANN201)